servers/mu/src/domain/index.js

```diff
@@ -30,6 +30,13 @@ const { DataItem } = warpArBundles
 const createDataItem = (raw) => new DataItem(raw)
 export { createLogger }
 
+const histogram = MetricsClient.histogramWith()({
+  name: 'outgoing_http_request_duration_seconds',
+  description: 'Outgoing request duration in seconds',
+  buckets: [0.5, 1, 3, 5, 10, 30],
+  labelNames: ['method', 'route', 'status', 'statusGroup']
```

Comment:
Won't `labelNames` need to have `operation` in it? Otherwise, stopping the timer will produce an error, not recording a metric and failing silently due to `swallow`.

Perhaps `withTimerMetrics` needs something such that it will "fail fast" if the consumer wants to set a label that is not allowed for the specific timer. 🤔  Otherwise, the seemingly benign action of adding a label downstream will break the metrics reporting. We'd want high, early, signal for that.

Reply:
Maybe for now, we add `operation` but put a comment on `withTimerMetrics` explaining as much. That labels reported _must_ be a subset of the labels defined on the `timer`.